backend/lib/endpoints/posts.js

```diff
@@ -368,6 +379,8 @@ async function routes(app) {
         ),
       );
       if (updateErr) {
+        throw app.httpErrors.internalServerError();
```

Comment:
```suggestion
        req.log.error(updateErr, "Failed liking post");
        throw app.httpErrors.internalServerError();
```

Reply:
I'll check if there are any more I'm missing. Thanks Josh!

Reply:
Update: I couldn't find any other instances of missing log.error messages.